### PR TITLE
Add GetKycSettings

### PR DESCRIPTION
**Pull request by subenson**

### CODE REVIEW

File: openapi/paths/kyc-settings.yaml [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: ../components/schemas/KycSettings.yaml
+    description: The settings for KYC analysis.
+    required: true
+  responses:
+    '200':
+      description: KYC settings were retrieved successfully.
```
Comment (noon-dawg) [inline openapi/paths/kyc-settings.yaml:20]: nitpick: avoid past tense
```suggestion
      description: KYC settings successfully retrieved .
```

**Comment by subenson** [comment]
@Rebilly/php @Rebilly/js please review

File: openapi/paths/kyc-settings.yaml [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+  tags:
+    - KYC Documents
+  summary: Get KYC settings
+  operationId: GetKycSettings
+  x-sdk-operation-name: getKycSettings
+  description: >
+    Retrieve KYC settings.
+  responses:
+    '200':
+      description: KYC settings successfully retrieved .
```
Comment (subenson) [inline openapi/paths/kyc-settings.yaml:13]:
```suggestion
      description: KYC settings successfully retrieved \.
```

Comment (subenson) [inline openapi/paths/kyc-settings.yaml:13]:
```suggestion
      description: KYC settings successfully retrieved.
```

**Comment by subenson** [comment]
@Rebilly/php @Rebilly/js please review

**Comment by subenson** [comment]
@Rebilly/php please review

**Review by XaosSintez — APPROVED** [review]

**Comment by subenson** [comment]
@Rebilly/php please review

**Review by colinbird — APPROVED** [review]

**Review by removed-account-123 — APPROVED** [review]

**Pull request merged**